plugins/modules/system/java_keystore.py

```diff
@@ -322,6 +348,14 @@ def read_stored_certificate_fingerprint(self):
 
         return stored_certificate_match.group(1)
 
+    def is_jks_or_pkcs12(self):
```

Comment:
```suggestion
    def current_keystore_type(self):
```
Nothing technically wrong with this, but the current function name sounds like a function outside of this class to which `self.keystore_path` would be passed as a parameter. Alternatively a simple name change without refactoring would make the method's purpose more clear IMO.

Reply:
oh yes, it sounds good. So I will do. Since it is in `JavaKeystore` class, maybe `current_type()` would be even better ?

Reply:
done